src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -44,13 +47,89 @@ const scrollTo = (div: null | HTMLDivElement, offset: number = 180) =>
     behavior: 'smooth',
   });
 
-interface ChartsHolderProps {
+interface RecommendationsProps {
   projections: Projections;
+  recommendationsRef: React.RefObject<HTMLDivElement>;
+}
+
+const Recommendations = ({
+  projections,
+  recommendationsRef,
+}: RecommendationsProps) => {
+  const alarmLevel = projections.getAlarmLevel();
+  const region = projections.region;
+  const projection = projections.primary;
+
+  const recommendationsIntro = getDynamicIntroCopy(
+    projection,
+    projections.locationName,
+    projections.getMetricValues(),
+  );
+
+  const recommendationsMainContent = getRecommendations(
+    projection,
+    mainContent.recommendations,
+  );
+
+  const recommendsShareUrl = getRecommendationsShareUrl(region);
+
+  const recommendsShareQuote = getShareQuote(
+    projections.locationName,
+    alarmLevel,
+  );
+
+  const recommendationsFeedbackForm = `https://can386399.typeform.com/to/WSPYSGPe#source=can&id=${uuidv4()}&fips=${
+    projection.fips
+  }`;
+
+  // TODO(Chelsi): make these 2 functions less redundant?
+  const recommendationsFedModalCopy = getModalCopyWithFedLevel(
+    projection,
+    projections.locationName,
+    projections.getMetricValues(),
+  );
+
+  const recommendationsHarvardModalCopy = getModalCopyWithHarvardLevel(
+    projection,
+    projections.locationName,
+    projections.getMetricValues(),
+  );
+
+  return (
+    <Recommend
+      introCopy={recommendationsIntro}
+      recommendations={recommendationsMainContent}
+      locationName={region.fullName}
+      shareUrl={recommendsShareUrl}
+      shareQuote={recommendsShareQuote}
+      recommendationsRef={recommendationsRef}
+      feedbackFormUrl={recommendationsFeedbackForm}
+      fedLevel={getFedLevel(projections.primary)}
+      harvardLevel={getHarvardLevel(projections.primary)}
+      harvardModalLocationCopy={recommendationsHarvardModalCopy}
+      fedModalLocationCopy={recommendationsFedModalCopy}
+    />
+  );
+};
+
+interface ChartsHolderProps {
   region: Region;
   chartId: string;
 }
-const ChartsHolder = ({ projections, region, chartId }: ChartsHolderProps) => {
-  const projection = projections.primary;
+
+const summaryToStats = (summary: LocationSummary): MetricValues => {
+  const stats = {} as MetricValues;
+  for (const metric of ALL_METRICS) {
+    stats[metric] = summary.metrics[metric]?.value ?? null;
+  }
+  return stats;
+};
+
+const ChartsHolder = ({ region, chartId }: ChartsHolderProps) => {
```

Comment:
now ChartsHolder doesn't need projections as a prop, so we can render it before projections are loaded.